<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_index</class>
 <widget class="QWidget" name="qt_index">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt_index</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>651</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="box_select_talbe">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_select_refresh">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>77</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="button_update_table_2">
      <property name="text">
       <string>更改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_delete_table_2">
      <property name="text">
       <string>删除记录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_create_table_2">
      <property name="text">
       <string>新建</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_insert_talbe_2">
      <property name="text">
       <string> 插入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttom_drop_table_2">
      <property name="text">
       <string>删除表</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>box_select_talbe</sender>
   <signal>activated(QString)</signal>
   <receiver>box_select_talbe</receiver>
   <slot>setEditText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
